ods_ci/tests/Tests/1100__data_science_pipelines/1110__test-run-data-science-pipelines-operator-e2e-tests.robot

```diff
@@ -27,26 +26,21 @@ Run Data Science Pipelines Operator Integration Tests
     ...     ODS-2632    AutomationBug
     ${openshift_api}    Get Openshift Server
     Log    ${openshift_api}
-    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DATA-SCIENCE-PIPELINES-OPERATOR-SDK_DIR} && make integrationtest K8SAPISERVERHOST=${openshift_api} DSPANAMESPACE=${DSPANAMESPACE} KUBECONFIGPATH=${KUBECONFIGPATH}
+    ${return_code}    ${output}    Run And Return Rc And Output    cd ${DSPO_SDK_DIR} && GIT_WORKSPACE=${DSPO_SDK_DIR} sh .github/scripts/tests/tests.sh --rhoai --k8s-api-server-host ${openshift_api} --kube-config ${KUBECONFIGPATH} --dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa.yaml --external-dspa-path ${DSPO_SDK_DIR}/tests/resources/dspa-external.yaml --clean-infra --endpoint-type route
```

Comment:
you sure about `.github/scripts/tests/tests.sh`? is that folder an hidden one given that it starts with "."?

Reply:
Yes. It is accurate. Initially, all scripts for DSPO were developed for GHA. We are making progress to improve. This script will probably end up in another directory in the near future.